### PR TITLE
Tickets/cap 720

### PR DESCRIPTION
**Pull request by craiglagegit**
New numbers for boresight and focus offsets

### CODE REVIEW

File: ATSpectrograph/v3/LATISS_AuxTel_20210318.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Ticket used to create this file: CAT-720
```
Comment (patrickingraham) [inline ATSpectrograph/v3/LATISS_AuxTel_20210318.yaml:1]: CAP-720

File: ATSpectrograph/v3/LATISS_AuxTel_20210318.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Date created: 2021-03-18
+# Description: Filter offset values based on measurements taken on 2021-01-08
+# Grating offsets assume focusing at filter central wavelengths
+# This new file includes grating pointing offsets measured on 2021-03-11
+# It also includes grating focus offsets for the ronchi170lpmm and holo4_003
+# measured on 2021-03-11.  The sign of these offsets is supect, but that's what
```
Comment (patrickingraham) [inline ATSpectrograph/v3/LATISS_AuxTel_20210318.yaml:7]: *suspect

**Review by patrickingraham — APPROVED** [review]
Please correct two typos (just amend the commit)
Note that this file has been validated.

**Comment by patrickingraham** [comment]
I also suggest rebasing so the history will appear as a single commit (instead of 3). Let me know if you need help.

**Comment by craiglagegit** [comment]
OK, I rebased to develop and corrected the two typos.  Ready to merge from my side.

**Comment by craiglagegit** [comment]
I rebased to develop and corrrected the two typos.  Ready to merge.
________________________________

**Comment by craiglagegit** [comment]
Merge done.

**Pull request merged**